kubernetes/templates/bigdl-distribute-workflow.yaml

```diff
@@ -0,0 +1,46 @@
+# BigDL Orca Distribute Workflow
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: bigdl-orca-ncf
```

Comment:
bigdl-ncf

Reply:
This is the job name, corresponding to `NOTES.txt` line 9. https://github.com/hamham223/BigDL/pull/29/files#diff-fa9d5f0c5a432abd99258bdde1949e496042eca73f8f81f55f56f457f211f158R9

Reply:
You need to modify notes as well.